doc_src/auto_examples/plot_fit_fake_data.ipynb

```diff
@@ -0,0 +1,90 @@
+{
+  "cells": [
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "collapsed": false
+      },
+      "outputs": [],
```

Comment:
what is this file?  -- why is it here, and does it show up in the documentation?

Reply:
This is needed for the sphinx gallery they need a py,rst.py.md5,ipynb, and zips for it to work